app/uk/gov/hmrc/lisaapi/controllers/Responses.scala

```diff
@@ -94,6 +99,10 @@ case object ErrorGenericBadRequest extends ErrorResponse(400, "BAD_REQUEST", "Ba
 
 case object ErrorAcceptHeaderInvalid extends ErrorResponse(406, "ACCEPT_HEADER_INVALID", "The accept header is missing or invalid")
 
+case object ErrorAcceptHeaderVersionInvalid extends ErrorResponse(406, "ACCEPT_HEADER_INVALID", "The accept header has an invalid version")
```

Comment:
This is new content that hasn't been reviewed yet - need to decide how we proceed with this

Reply:
We've updated this